src/main/java/browserview/ChromeDriverEx.java

```diff
@@ -50,9 +52,7 @@ public void get(String url, boolean isForceRefresh) throws WebDriverException {
             logger.info("Test loading page: " + url);
             testGet();
         } else {
-            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url) ||
-                driver.getCurrentUrl().equalsIgnoreCase(url + "/commits") ||
-                driver.getCurrentUrl().equalsIgnoreCase(url + "/files"))) {
+            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url))) {
```

Comment:
TL:DR
As long as `get` stays the way it was, it's impossible to navigate to the `files` and `commits` PR tabs without adding a new method to `ChromeDriverEx`